### ISSUE TITLE
trouble trying to get FilePondPluginFileValidateSize working

### ISSUE THREAD

**Issue by kmanley** [pqina/filepond#12]
Thank you for creating filepond, I found the integration really easy and it looks great. 

I'm having a little problem trying to get the file size validator plugin working though -- the following code from docs doesn't seem to work, I get an error about setDefaults not existing. Any help would be appreciated; I'm restricting filesize on the server but the UI experience will be better if I can get this working. Thanks!

```
$(function(){

    $.fn.filepond.registerPlugin(FilePondPluginFileValidateSize);

    $.fn.filepond.setDefaults({
        maxFileSize: '3MB'
    });

});```

**Comment by rikschennink** [comment]
Hm, seems to be a problem with the `setDefaults` / `setOptions` method. When assigning the `'3MB'` value like this, it works.

```js
$('.my-pond').filepond('maxFileSize', '3MB');
```

I'm looking into it.

**Comment by rikschennink** [comment]
The problem was the `'3MB'` string not being converted properly to bytes when setting it as a default option. Fixed the issue and just published version `1.2.4`. You can leave the jQuery adapter as is.
